lib/seadsa/DsaCallSite.cc

```diff
@@ -18,7 +18,7 @@ bool DsaCallSite::isPointerTy::operator()(const Argument &a) {
 
 static const Function *
 getCalledFunctionThroughAliasesAndCasts(const CallBase &cb) {
-  const Value *CalledV = cb.getCalledValue();
+  const Value *CalledV = cb.getCalledOperand();
```

Comment:
is `getCalledOperand` the new name of `getCalledValue`? I seem to recall using `getCalledValue` even with `llvm::CallBase` in llvm10

Reply:
I think so. In LLVM 11, the `CallBase` class doesn't have a `getCalledValue` method.

Reply:
This is what I found in LLVM 10's definition of `CallBase`.

```C++
  // DEPRECATED: This routine will be removed in favor of `getCalledOperand` in
  // the near future.
  Value *getCalledValue() const { return getCalledOperand(); }
```

Reply:
thanks. this answers my question